generated/google_cloud_logging_type/.sidekick.toml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[general]
+specification-source = 'google/logging/type'
+
+[source]
+title-override = "Logging types"
```

Comment:
[nit] I can't tell if there is a reason behind which strings are single vs double quoted - it doesn't look like any of the double quoted strings are using escape characters. Consider using `'` consistently.